### ISSUE TITLE
Documentation troubleshooting: browser rejects vicadmin and Management Portal certificate

### ISSUE THREAD

**Issue by hickeng** [vmware/vic-product#526]
**Problem**
Chrome (and maybe other browsers) report `ERR_CERT_INVALID` and the detail reports an "invalid digital signture"

**Cause**
Probable cause is that a client certificate or generated CA for a prior VCH with the same IP or hostname was added to the client computers keychain.

**Solution**
Windows 10: run the `Manage user certificates` utility and ensure no Certificate Authorities or client certificates are installed for the target VCH IP or FQDN. These will likely be found under either of the following sections:
* `Personal`
* `Trusted Root Certification Authorities`

Generated VCH certificates have the VCH name (of the old VCH) as the Issuer.

**Comment by stuclem** [comment]
This is also the case for attempts to access VIC Management Portal on Chrome. Clearing certificates out of Personal and Trusted Root Certification Authorities did not solve the problem for me for the VIC MP. 

Searching through the other certificate folders, I found a bunch of VMware self-signed certs in the following places:

![oldcert](https://user-images.githubusercontent.com/16718369/29816395-359fec0a-8cb4-11e7-931d-cb5fa861afdd.png)


![oldcert2](https://user-images.githubusercontent.com/16718369/29816392-359bd160-8cb4-11e7-843b-2bb830420c68.png)


![oldcert3](https://user-images.githubusercontent.com/16718369/29816393-359c2c6e-8cb4-11e7-96ef-5b872992c970.png)


![oldcert4](https://user-images.githubusercontent.com/16718369/29816394-359f0dd0-8cb4-11e7-8e7e-0dc262e72a35.png)

It was only when I deleted all of the self-signed certs in the Intermediate Certification Authorities folder that I was able to access VICMP once again.
